src/translations/pt/core.js

```diff
@@ -0,0 +1,30 @@
+export default {
+  heroTitle: "Construa sistemas de design melhores",
+  heroSubtitle:
+    "Uma lista de verificação de código aberto para ajudá-lo a planejar, construir e expandir seu sistema de design.",
```

Comment:
Same thing here with "lista de verificação", there is no nees to translate the expression "checklist". You can use the english expression or say just "lista".

Reply:
It's already been fixed.